website/docs/r/is_instance.html.markdown

```diff
@@ -338,7 +339,7 @@ Review the argument references that you can specify for your resource.
 
 ## Attribute reference
 In addition to all argument reference list, you can access the following attribute reference after your resource is created.
-
+- `availability_policy_host_failure` - (String) The availability policy to use for this virtual server instance. The action to perform if the compute host experiences a failure. 
```

Comment:
availability_policy_host_failure is a argument no need to be presen in Attribute Refrence

Reply:
removed